services/nvidia-gpu-operator/1.11.1/defaults/cm.yaml

```diff
@@ -5,12 +5,22 @@ metadata:
   name: nvidia-gpu-operator-1.11.1-d2iq-defaults
   namespace: ${releaseNamespace}
 data:
-  values.yaml: |-
+  values.yaml: |
     nfd:
       enabled: false
     driver:
       enabled: false
     toolkit:
-      version: v1.10.0-centos7
+      version: v1.11.0-ubuntu20.04
     gfd:
       enabled: true
+    dcgm:
+      enabled: true
+      version: 2.3.6-1-ubuntu20.04
```

Comment:
just out of curiosity, are these versions related to the underlying host OS? aka will they change depending on the OS being used in the cluster?

Reply:
yes it will! thank you this comment. i added notes explaining why the values were chosen.

Reply:
perfect, thanks!